docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,321 @@ redirect_from:
+
+---
+
+### Sending and responding to actions
+
+To use features like buttons, select menus, datepickers, dialogs, and shortcuts, you’ll need to enable interactivity. Similar to events, you'll need to specify a URL for Slack to send the action (such as *user clicked a button*).
+
+Back on your app configuration page, click on **Interactivity & Shortcuts** on the left side. You'll see that there's another **Request URL** box.
+
+By default, Bolt is configured to use the same endpoint for interactive components that it uses for events, so use the same request URL as above (in the example, it was `https://8e8ec2d7.ngrok.io/slack/events`). Press the **Save Changes** button in the lower right hand corner, and that's it. Your app is set up for interactivity!
+
+![Configuring a Request URL](../assets/request-url-config.png "Configuring a Request URL")
+
+Now, let's go back to your app's code and add interactivity. This will consist of two steps:
+- First, your app will send a message that contains a button.
+- Next, your app will listen to the action of a user clicking the button and respond
+
+Below, I've modified the app code we wrote in the last section to send a message with a button rather than a string:
+
+```python
+import os
+from slack_bolt import App
+
+# Initializes your app with your bot token and signing secret
+app = App(
+    token=os.environ.get("SLACK_BOT_TOKEN"),
+    signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+# Listens to incoming messages that contain "hello"
+@app.message("hello")
+def message_hello(message, say):
+    # say() sends a message to the channel where the event was triggered
+    say(
+        blocks=[
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": f"Hey there <@{message['user']}>!"
+                },
+                "accessory": {
+                    "type": "button",
+                    "text": {
+                        "type": "plain_text",
+                        "text": "Click Me"
+                    },
+                    "action_id": "button_click"
+                }
+            }
+        ],
+        text=f"Hey there <@{message['user']}>!"
+    )
+
+# Start your app
+if __name__ == "__main__":
+    app.start(port=int(os.environ.get("PORT", 3000)))
+```
+
+<!-- TODO - Review my usage of "List" (Array?) and clarification of the role of `text` -->
+The values passed to `say()` are now a list of `blocks` and a string of `text`. Blocks are the building components of a Slack message and can range from text to images to datepickers. In this case, your app will respond with a section block that includes a button as an accessory. Since we're using `blocks`, the `text` is a fallback string to display in notifications.
```

Comment:
🆘  Python expertise needed!
- In the first sentence, I'd appreciate a review of my usage of `List` (or should be it `Array`)
- In the last sentence, I'd appreciate a review of my clarification of the role of the fallback `text` argument

Reply:
- Yep! `list` is the correct data type. I might consider an adjustment to the first sentence: 

> The arguments passed to `say()` now include a list of `blocks` and a string of `text`. 

I'm not sure in this specific case how helpful it is to point out to the dev what the exact data types are, as they can see them, but just a passing thought!

- I notice that we don't specify this in the BoltJS documentation. Should we? If I understand this right, it will show the value of `text` only for notification purposes, but in the actual client, the blocks will appear? If that's accurate, a potential rewrite (and if not, disregard entirely):

> The value provided for `text` is the message displayed in notifications, since blocks can only be rendered in the client window.

Reply:
We don't explicitly state where Slack will and will not use `text` once `blocks` are supplied. But as of now, it is used in notifications (where blocks cannot be rendered anyway) and I _think_ its available to screen readers. It _might_ be worth stating that its always recommended to use `text` for accessibility purposes.

Reply:
Thanks for the discussion @misscoded and @aoberoi!

In [this conversation with @shaydewael](https://github.com/slackapi/bolt-python/pull/71#discussion_r488229309), we decided to revert the paragraph to align with `bolt-js`. That will clear up the awkwardness around `List` and `text`.  As a closing sentence, we mention that `text` can be used as a fallback for notifications and accessibility.

> The value inside of `say()` is now an object that contains an array of `blocks`. Blocks are the building components of a Slack message and can range from text to images to datepickers. In this case, your app will respond with a section block that includes a button as an accessory. Since we're using `blocks`, the `text` is a fallback for notifications and accessibility.

I'll make a matching update in `bolt-js`.